love/src/Config.js

```diff
@@ -702,7 +702,23 @@ export const m1m3ILCStateToStyle = {
   OFF: 'warning',
 };
 
-// MTM1M2
+// MTM2
+export const m2PowerTypeStateToStyle = {
+  UNKNOWN: 'invalid',
+  ACTUATOR: 'ok',
+  ILC: 'ok',
+};
+
+export const m2PowerStateToStyle = {
+  UNKNOWN: 'invalid',
+  INITIALIZATION: 'ok',
+  ON: 'ok',
+  OFF: 'warning',
```

Comment:
Why `OFF` is `warning`? Do we have other option such as `off`?

Reply:
we have and we use the `warning` status for all of those, which is a yellowish status. There is a palette with the colors we uses for statuses on https://summit-lsp.lsst.codes/love/palette (status color section).